modules/search/instant-search/components/path-breadcrumbs.scss

```diff
@@ -0,0 +1,22 @@
+.jetpack-instant-search__path-breadcrumb {
```

Comment:
Do we need to include this stylesheet somewhere? I'm not seeing these styles applied:

<img width="326" alt="Screen Shot 2020-10-27 at 17 13 49" src="https://user-images.githubusercontent.com/17325/97256925-3e48cd00-1879-11eb-93f1-6d53909390f9.png">

Reply:
Whoops, I botched the stylesheet import with a typo. This has been fixed.